<launch>
  <arg name="uav_id_"  />
 <arg name="pixel_ratio_"  />
  <arg name="max_pose_error"  />
  <arg name="drone_width"  />
  <arg name="drone_height"  />
  <!-- Launch ROS Package Template Node -->
  <node pkg="uav_detect" type="uav_detect" name="uav$(arg uav_id_)_detect" output="screen">
      <remap from="~depth"       to="/uav_$(arg uav_id_)_pcl_render_node/depth"/>
      <remap from="~odometry"        to="/uav_$(arg uav_id_)_global_odom"/> 
      <remap from="~new_depth_image"        to="/uav_$(arg uav_id_)_new_depth_image"/> 
      <remap from="~others_odom"        to="/others_odom"/> 
      
          <!-- 相机参数信息 -->
        <param name="cam_cx" value="321.04638671875"/>
        <param name="cam_cy" value="243.44969177246094"/>
          <param name="cam_fx" value="387.229248046875"/>
        <param name="cam_fy" value="387.229248046875"/>
        <param name="cam_width" value="640"/>
        <param name="cam_height" value="480"/>

        <param name="pixel_ratio" value="$(arg pixel_ratio_)"/>
        <param name="estimate/max_pose_error" value="$(arg max_pose_error)"/>
        <param name="estimate/drone_width" value="$(arg drone_width)"/>
        <param name="estimate/drone_height" value="$(arg drone_height)"/>

        <param name="my_id"      value="$(arg uav_id_)" />

  </node>

</launch>